index.html

```diff
@@ -45,30 +55,29 @@ <h1 class="greetings">Hey There. I’m Miguel</h1>
     </div>
   </section>
   <section class="works">
-    <ul>
+    <ul class="works-grid">
       <li class="works-title">
-        <div>
+        <div class="works-title-container">
           <h1 class="recent-works-title">My Recent Works</h1>
           <div class="recent-works-line"></div>
         </div>
       </li>
       <li class="multi-post-stories">
         <div class="multi-post-stories-image">
-          <img src="images/multi-post-stories.svg" alt="background image">
         </div>
-        <div>
+        <div class="multi-post-stories-container">
           <h1 class="multi-post-stories-title">Multi-Post Stories</h1>
           <p class="multi-post-stories-content">A daily selection of privately
             personalized reads; no accounts or sign-ups required. has been the
             industry's standard dummy text ever since the 1500s, when an unknown
             printer took a standard dummy text.
           </p>
           <div class="multi-post-stories-languages">
-            <ul>
-              <li>css</li>
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
+            <ul class="languages">
```

Comment:
- [ ] [optional] Kindly check the design of your mobile version, it broken check the comparison below:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/187034728-cafaaa30-c111-48cd-a6a8-f9847c1d2872.png) |  ![image](https://user-images.githubusercontent.com/61408860/187034760-f5679c4a-d4b4-43e0-b181-6dd5dd7f5473.png)